nilearn/tests/test_ndimage.py

```diff
@@ -21,6 +25,48 @@ def test_largest_cc():
     b[5, 5, 5] = 1
     np.testing.assert_equal(a, largest_connected_component(b))
 
+    # Tests for correct errors, when an image or string are passed.
+    img = testing.generate_labeled_regions(shape = (10,11,12),
+                                           n_regions = 2 )
+
+    assert_raises(ValueError, largest_connected_component, img)
+    assert_raises(ValueError, largest_connected_component, "Test String")
+
+
+def test_largest_cc_img():
+    """ Check the extraction of the largest connected component, for niftis
+
+    Similiar to smooth_img tests for largest connected_component_img, here also
+    only the added features for largest_connected_component are tested.
+    """
+
+    #Test whether dimension of 3Dimg and list of 3Dimgs are kept.
+    shapes = ((10, 11, 12), (13, 14, 15))
+    regions = [1,3]
+
+    img1 = testing.generate_labeled_regions(shape = shapes[0],
+                                            n_regions =regions[0] )
```

Comment:
Here too, PEP8 unexpected white space between n_regions and = and space before ")"